### ISSUE TITLE
Custom spinner

### ISSUE THREAD

**Issue by JonathanVandal** [miromannino/Justified-Gallery#231]
It's possible to custom the spinner when we are loading the images ? I'd like to had material spinner

**Comment by miromannino** [comment]
Yes it is quite easy to customize that part, even if you should branch the code in order to do that

I suggest something light like this: https://codepen.io/mrrocks/pen/EiplA

Unfortunately I am not considering to change it, because, especially for infinite scrolling, the spinner needs to be a little bit short since it is at the end of a page.

Depending on what you are doing, if you have the images' width and height I always suggest [waitThumbnailsLoad = false](http://miromannino.github.io/Justified-Gallery/performance-tips/)

**Comment by JonathanVandal** [comment]
Branch the code ? Do you mean that I have to edit your library ?

Yeah I have already waitThumbnailsLoad = false and I have the same material spinner
